src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -579,6 +567,11 @@ h2::before {
     padding: 0.75rem 0 !important;
   }
 
+  // bring the main label down a bit to line up with text-field
+  .row:first-of-type .col:first-of-type {
+    padding-top: 8px !important;
+  }
```

Comment:
This is unique to the Certify component, since there is a text-field (box) at the top of the second column. This change looks better (and matches the comps).